docs/CODE_STYLE.md

```
@@ -45,12 +45,12 @@
 ### Constants
 
 **Macros**:
-- To denote their group, use a short prefix in all caps (e.g., `PREFIX_`).
-- The rest should be in all caps `SNAKE_CASE`.
+- Written in all caps `SNAKE_CASE`.
+- Try to denote their group using a short prefix (e.g., `PREFIX_`).
 - Example: `#macro COL_DARK_RED`.
 
 **Enums**:
-- Enum names should start with an `e` prefix and be in all caps.
+- Enum names should start with an `e` prefix and be in all caps `SNAKE_CASE`.
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)